storage-proofs-core/src/merkle/mod.rs

```diff
@@ -2,9 +2,9 @@
 
 use std::fs::File;
 
+pub use filecoin_hashers::Hasher;
```

Comment:
Is this export being used anywhere in this PR? I'm not sure that I see it. Maybe it's being re-exported because it's part of the public Merkle tree API in `storage-proofs-core`?

Reply:
It's explained in the commit message https://github.com/filecoin-project/rust-fil-proofs/pull/1745/commits/c92510e2029188f0ec88cc50e96db0b7410ac4b8:

> The `filecoin_hashers::Hasher` trait is needed a lot in downstream dependants when the domain of the Tree hasher needs to be accessed. Hence re-export it, so that those dependents don't need to depend directly on `filecoin_hashers` only due to this single trait.